src/chart/Sankey.tsx

```diff
@@ -429,6 +432,13 @@ export class Sankey extends PureComponent<Props, State> {
     links: [] as SankeyLink[],
   };
 
+  wrapperAriaLabel: string;
+
+  constructor(props: Props) {
+    super(props);
+    this.wrapperAriaLabel = props.wrapperAriaLabel ?? uniqueId('Recharts Sankey Chart ');
```

Comment:
why do we need a constructor here (and all others in PR with same approach)? Its probably more simple and just as efficient to reference props directly

Why is there a space in all of the uniqueId calls? I also don't think these need to be unique?

Reply:
They do need to be unique; that's the motivation for the PR. 

> Region
>
> A [region](https://w3c.github.io/aria/#region) landmark is a perceivable section of the page containing content > that is sufficiently important for users to be able to navigate to the section.
> 
> - A region landmark must have a label.
> - If a page includes more than one region landmark, each should have a unique label (see [Step 3](https://www.w3.org/WAI/ARIA/apg/practices/landmark-regions/#aria_lh_step3) above).
> - The region landmark can be used identify content that named landmarks do not appropriately describe.

(from [https://www.w3.org/WAI/ARIA/apg/practices/landmark-regions/#aria_lh_region](https://www.w3.org/WAI/ARIA/apg/practices/landmark-regions/#aria_lh_region))

The space is there so that screen readers can pronounce the name.

Reply:
I would like to understand this better - how does the space help screen readers? 
Without a comment to explain this in the code, I am afraid the next developer might remove the space.

Reply:
Should we possibly extract a helper function with an intention revealing name, such as:

```
// Some useful reference
function addTrailingSpaceForScreenReaders(label: string){
   return label+' '
}
```

Reply:
+1, that space will get removed in a heartbeat by someone who thinks it was an accident. Would be better if we can wrap it somehow